CrewChiefV4/GameDataReader.cs

```diff
@@ -45,10 +44,20 @@ public Boolean Initialise()
                 catch (Exception)
                 {
                     Console.WriteLine("Unable to create folder for data file, no session record will be available");
-                    dumpToFile = false;
+                    dataFilesPath = null;
                 }
             }
+        }
+
+        public Boolean Initialise()
+        {
+            Console.WriteLine("initialising");
```

Comment:
Tested now, works.  The only issue is this "initialising" message will get fired many times when process has started, but mmf not yet available.  In past it was on line 63.  Maybe we should name it "Attempting to connect to the game..." or something.